README.md

```diff
@@ -256,12 +268,46 @@ Valid values are 'true', 'false', 'manual'.
 
 `binary`, `control`, `hasrestart`, `hasstatus`, `manifest`, `path`, `pattern`, `provider`, `restart`, `start`, `status`, `stop`
 
+## `types::mailalias`
+No helper resources are implemented. Simply passes attributes to a mailalias resource.
+
+### mostly used parameters
+
+ensure
+------
+Whether a service should be running.
+
+Valid values are 'present' and 'absent'.
+
+- *Default*: 'present'
+
+provider
+--------
+Whether a service should be enabled to start at boot.
+Valid values are 'true', 'false', 'manual'.
+
+- *Default*: 'aliases'
+
+recipient
+---------
+Whether a service should be enabled to start at boot.
+Valid values are 'true', 'false', 'manual'.
+
+- *Default*: 'MANDATORY'
+
+target
+------
+Whether a service should be enabled to start at boot.
+Valid values are 'true', 'false', 'manual'.
+
+- *Default*: '/etc/aliases'
+
 ===
 
 # Hiera
 
 ## cron
-<pre>
+```
```

Comment:
This is fine, though in the future please separate PR's with new functionality from those that are doing a bug fix or formatting change. That makes it much easier to get them both merged.